source/upgrade/upgrading-mattermost-server.rst

```diff
@@ -165,7 +237,14 @@ Upgrading Mattermost Server
 
     sudo systemctl start mattermost
 
-12. If you're using a `High Availability <https://docs.mattermost.com/scale/high-availability-cluster.html>`__ deployment, you need to apply the steps above on every node in your cluster. Once complete, the **Config File MD5** columns in the High Availability section of the System Console should be green. If they're yellow, please ensure that all nodes have the same server version and the same configuration.
+12. Remove the temporary files.
+
+  .. code-block:: sh
+
+    sudo rm -r /tmp/mattermost-upgrade/
```

Comment:
We need to remove lines 208/209 now... they are not needed now that we have Section 12

Reply:
You're absolutely right, @sadohert! Thanks!